package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "stylelint-create",
+  "version": "0.0.1",
+  "description": "Initialization wizard for Stylelint, a linter that helps you avoid errors and enforce conventions in your styles.",
+  "homepage": "https://github.com/stylelint/stylelint-create#readme",
+  "bugs": {
+    "url": "https://github.com/stylelint/stylelint-create/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stylelint/stylelint-create.git"
+  },
+  "license": "MIT",
+  "author": "stylelint",
+  "main": "create-stylelint.mjs",
+  "bin": {
+    "create-stylelint": "./create-stylelint.mjs"
+  },
+  "scripts": {
+    "format": "prettier . --write --cache",
+    "jest": "jest",
+    "lint": "npm-run-all --parallel --continue-on-error lint:*",
+    "lint:formatting": "prettier . --check --cache",
+    "lint:js": "eslint . --cache --max-warnings=0 --ext .js,.mjs",
+    "lint:md": "remark . --quiet --frail",
+    "test": "vitest",
+    "watch": "vitest"
+  },
+  "prettier": "@stylelint/prettier-config",
+  "eslintConfig": {
+    "extends": [
+      "stylelint"
+    ],
+    "globals": {
+      "__dirname": true,
+      "module": true,
+      "require": true,
+      "testRule": true
+    },
+    "root": true
+  },
+  "remarkConfig": {
+    "plugins": [
+      "@stylelint/remark-preset"
+    ]
+  },
+  "dependencies": {
+    "chalk": "^5.2.0",
+    "execa": "^6.1.0",
+    "ora": "^6.1.2",
+    "which-pm-runs": "^1.1.0"
+  },
+  "devDependencies": {
+    "@stylelint/prettier-config": "^2.0.0",
+    "@stylelint/remark-preset": "^4.0.0",
+    "eslint": "^8.28.0",
+    "eslint-config-stylelint": "^17.1.0",
+    "jest": "^28.1.3",
+    "jest-preset-stylelint": "^5.0.4",
```

Comment:
[suggest] Jest is unnecessary.

```suggestion
```

Reply:
Ah, so this is what I noted above in the PR description; if I remove this, e.g. [see this commit's action run](https://github.com/stylelint/stylelint-create/actions/runs/4038050780/jobs/6941775169), then `eslint-plugin-jest` fails because `jest` doesn't exist. 

I assume the path forward is to disable the plugin, but "unextending" seems trickier than I thought. Any suggestions?

Reply:
One option is to set the Jest version in `package.json`:

```
"settings": {
    "jest": {
      "version": 26
    }
 }
```

It also appears this issue only affects the ESLInt `jest/no-deprecated-functions` rule

Discussion of this can be seen here https://github.com/jest-community/eslint-plugin-jest/issues/686

An additional suggestion is to use `latest` as the version ,if `latest` works this would be far more maintainable and we wouldn't have to update this ourselves with each new Jest major release

https://github.com/jest-community/eslint-plugin-jest/issues/686#issuecomment-739151588
> A workaround for this issue is to specify the version via `latest `as most ESLint versions are back-wards compatible in the `.eslintrc`.

Reply:
Oh, our ESLint config should be updated to exclude the configs for Jest:

https://github.com/stylelint/eslint-config-stylelint/blob/43802c04eef4134680a58205271c8111a05f0434/.eslintrc.js#L16-L17

The `jest/no-deprecated-functions` rule may require that the Jest package should be installed, so could you disable only the rule to suppress the error like this?

```diff
--- a/package.json
+++ b/package.json
@@ -46,6 +46,9 @@
     "extends": [
       "stylelint"
     ],
+    "rules": {
+      "jest/no-deprecated-functions": "off"
+    },
     "root": true
   },
```

Reply:
Sorry, I missed @ntwb's comment before I posted. It also seems reasonable to set `settings.jest.version` as a workaround.

I've opened an issue related to this problem: https://github.com/stylelint/eslint-config-stylelint/issues/205

Reply:
Ended up setting to `latest` for now; happy to resolve later once we have a more permanent solution!